### DISCUSSION TITLE
Open pose color scheme for Hands and Feet

### DISCUSSION THREAD

**Discussion by if-ai** [Mikubill/sd-webui-controlnet#389] [Q&A]
Hi, first thank you for developing this extension is great. I have some doubts and questions I need help with I am developing a free rig for Maya and Blender and I notice you deleted the open pose hands processor. I know is because the production version doesn't support it but my question is if you will be adding it in the future and if you could share the colors scheme you using so I can make them in the meantime so it will be ready for users when you eventually release it? Thank you.

**Comment by ljleb** [comment]
Hi, you can use the official openpose documentation as a reference:
https://cmu-perceptual-computing-lab.github.io/openpose/web/html/doc/md_doc_02_output.html#body-keypoints-in-c-python

It has colors of both for hands and body. maybe it is possible to use a color picker to approximate color codes?

**Reply by if-ai** [reply]
Thank you, I have the body points already but going to check this documentation if it has the hands and you are following the same convention then it should work.

**Reply by if-ai** [reply]
Yes this is what I needed thanks a lot
